package.json

```diff
@@ -1,29 +1,46 @@
 {
-  "name": "kilt-extension-api",
+  "name": "@kiltprotocol/kilt-extension-api",
   "version": "0.1.0",
-  "main": "./index.js",
-  "types": "./index.d.ts",
-  "type": "commonjs",
+  "main": "./dist/cjs/index.js",
+  "types": "./dist/cjs/index.d.ts",
+  "type": "module",
   "exports": {
-    ".": {
-      "types": "./index.d.ts",
-      "default": "./index.js"
+    "." : {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js",
+      "default": "./dist/cjs/index.js"
     },
-    "./wellKnownDidConfiguration": {
-      "types": "./wellKnownDidConfiguration/index.d.ts",
-      "default": "./wellKnownDidConfiguration/index.js"
+    "./wellKnownDidConfiguration" : {
+      "import": "./dist/esm/wellKnownDidConfiguration/index.js",
+      "require": "./dist/cjs/wellKnownDidConfiguration/index.js",
```

Comment:
I think the cjs build should still live in the root, like it did before, for tools that don't understand the exports map yet

Reply:
cjs build is (thanks 2 u) in the root now :cry:  : [ea72411](https://github.com/KILTprotocol/kilt-extension-api/pull/27/commits/ea7241148c61bc9db532cb40fc3f4054013cfb3a).